blueprint/security/hybrid-security-risk-management-plan.md

```diff
@@ -1290,13 +1290,13 @@ Administration of the system is undertaken by authorised privileged users by con
 
 2 - Medium
 
-### R17 Mobile device compromised
+### R17 Mobile phone compromised due to loss or theft
 
 #### Risk overview
 
-An Apple iOS device used to access Sensitive and or security classified data is compromised as a result of the [ACSC’s Security Configuration Guide - Apple iOS 12 Devices](https://www.cyber.gov.au/publications/security-configuration-guide-apple-ios-12-devices) (September 2019)  not being fully implemented due to the usability impacts. 
+An Apple iOS device used to access Sensitive and or security classified information is compromised as a result of the device being lost or stolen.
 
-Note, the HybridSystem does not include the use of devices using the Android operating system.
+Note: the CloudSystem does not include the use of devices using the Android operating system.
```

Comment:
`CloudSystem` should be `HybridSystem`. There are 2 more references in this document.

Reply:
Updated